GameServerLib/GameObjects/AttackableUnits/AI/ObjAIBase.cs

```diff
@@ -967,6 +966,47 @@ public override void Update(float diff)
             }
         }
 
+        public override void TakeDamage(IAttackableUnit attacker, float damage, DamageType type, DamageSource source, DamageResultType damageText)
+        {
+            base.TakeDamage(attacker, damage, type, source, damageText);
+            OnTakeDamage(attacker);
+        }
+        void OnTakeDamage(IAttackableUnit attacker)
```

Comment:
Remove the unnecessary extra TakeDamage function and place the body inside the TakeDamage above.

Reply:
Not yet possible, because a second `TakeDamage` function has appeared